plugins/node/opentelemetry-instrumentation-pg/test/pg-pool.test.ts

```diff
@@ -563,6 +563,7 @@ describe('pg-pool', () => {
             0,
             'expected to have 0 pending requests'
           );
+          done();
```

Comment:
Ah, this is a clearer fix.
I have a similar fix for this test failure here: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2464/files?diff=unified&w=1#diff-682483846290c7e14fba934e5f5effaaa2ff8e84e94f511300af6d0120a59251R495-R570
as part of a larger PR